docs/source/gbd2019_models/hemoglobin_anemia_and_iron_deficiency/risk_exposure/index.rst

```diff
@@ -7,3 +7,172 @@ Iron Deficiency Risk Exposure
+
+	This exposure value is stored as the exposure for the iron deficiency risk factor in GBD 2019.
+
+	The iron deficiency risk exposure estimated in this way can be interpreted as the counterfactual population mean hemoglobin concentration in the absence of all diseases that do NOT cause iron deficiency, according to a correspondence with Nick Kassebaum, who leads the GBD team responsible for these estimates. However, it is important to note that the code used does not consider diseases that cause non-dietary iron deficiency.
+
+.. list-table:: MEIDs Used for the Global TMREL exposure estimation
+  :widths: 15
+  :header-rows: 1
+
+  * - MEIDs
+  * - 23376, 23381, 23386, 23377, 23382, 23387, 23378, 23383, 23388, 23379, 23384, 23389, 1929, 1930, 1931, 1925, 1926, 1927, 2131, 2132, 2133, 2065, 2066, 2067, 2082, 2083, 2084, 2113, 2114, 2115, 2506, 2507, 2508, 2119, 2120, 2121, 2492, 2493, 2494, 2495, 2496, 2497, 2498, 2499, 2500, 2502, 2503, 2504, 2475, 2476, 2477, 2478, 2479, 2480, 2481, 2482, 2483, 2485, 2486, 2487, 2489, 2490, 2491, 1666, 1667, 1668, 19391, 19392, 19393, 19395, 19396, 19397, 19399, 19400, 19401, 16314, 16315, 16316, 1538, 1539, 1540, 1522, 1523, 1524, 1532, 1533, 1534, 1476, 1477, 1478, 19737, 19738, 19739, 19798, 19799, 19800, 18864, 18865, 18866, 18861, 18862, 18863
+
+.. note:: 
+
+	The MEIDs listed above were obtained from the *"out_meids"* tab in the excel file hosted `here <https://stash.ihme.washington.edu/projects/MNCH/repos/anemia_causal_attribution/browse/in_out_meid_map.xlsx>`_.
+
+Location-Specific TMREL Exposure Estimation
+++++++++++++++++++++++++++++++++++++++++++++
+
+This estimation approach is described in the GBD 2019 Risk Factor Methods Appendix, but NOT used in GBD 2019 iron deficiency risk exposure calculations. Therefore, the following description does not reflect the values stored for the GBD 2019 iron deficiency risk factor, but represent an alternative estimation approach that is "conceptually/clinically more correct" than the Global TMREL exposure estimation approach according to a correspondence with Nick Kassebaum, although it did not meet the GBD 2019 requirement that TMRELs are not location-specific.
+
+The location-specific TMREL exposure estimation approach has the following steps:
+
+#. Estimate a location-specific TMREL, defined as the counterfactual population mean hemoglobin concentration in the absence of all diseases that cause iron deficiency
+
+	:math:`TMREL = mean_\text{Hgb} + \sum_{c=1}^{n} p_c * shift_c`
+
+	Where, :math:`\sum_{c=1}^{n} p_c * shift_c` represents the sum of the cause-specific prevalence multiplied by the cause-specific hemoglobin shift for all causes of iron deficiency and :math:`mean_\text{Hgb}` is the observed mean population hemoglobin concentration (MEID 10487).
```

Comment:
Looks like this (and the equation above) is indented farther than the text after #. in your first list item, so it shows up as a blockquote rather than just continuing the preceding paragraph. Not sure if that's what you intended.